client/connection.go

```diff
@@ -32,6 +32,10 @@ type ConnectionArgs struct {
 
 	// Custom HTTP Client (used as base for the connection)
 	HTTPClient *http.Client
+
+	// Controls whether a client verifies the
+	// server's certificate chain and host name.
```

Comment:
tiny nitpick, but can you keep the comment on a single line :)

Reply:
yes, that's better